docs/en/latest/get-data-planes-from-etcd.md

```diff
@@ -0,0 +1,97 @@
+---
+id: get-data-planes-from-etcd
+title: Get data planes from etcd
+keywords:
+  - API gateway
+  - Apache APISIX
+description: Get data planes from etcd
+---
+
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+To query all data planes information from etcd, we move `server-info` plugin into the APISIX core, and use it to report data plane information.
+
+The data plane information keeps the same format of `server-info` plugin output:
+
+https://apisix.apache.org/en/docs/apisix/plugins/server-info/
+
+## Admin API to query data planes
```

Comment:
We should move this into Admin API doc: https://github.com/apache/apisix/blob/master/docs/en/latest/admin-api.md